torchao/dtypes/fpx/fpx.py

```diff
@@ -347,101 +354,81 @@ def from_scaled_tc_fpx(tensor: Tensor, ebits: int, mbits: int, scale=None) -> Te
 ]
 
 
-class QuantLlmLinearWeight(Tensor):
+class FpxTensor(Tensor):
+    """FpxTensor represents a Tensor with dtype fpx(ebits=a, mbits=b),
+    it has a internal tensor field of "packed_fpx_data", which is packed from the
+    uint8 unpacked data (the output of `quantize_affine_fpx` operator)
+    """
     implements = classmethod(_implements)
     __torch_function__ = classmethod(_dispatch__torch_function__)
     __torch_dispatch__ = classmethod(_dispatch__torch_dispatch__)
 
     @staticmethod
-    def __new__(cls, fpx_data: Tensor, scale: Tensor, ebits: int, mbits: int):
-        assert fpx_data.ndim == 2
-        assert fpx_data.dtype == torch.uint8
-        shape = (fpx_data.shape[0], fpx_data.shape[1] // (1 + ebits + mbits) * 8)
+    def __new__(cls, packed_fpx_data: Tensor, ebits: int, mbits: int):
+        assert packed_fpx_data.ndim == 2
+        assert packed_fpx_data.dtype == torch.uint8
```

Comment:
maybe add some docs on the expected layout of the packed data

Reply:
added